MainWindow.xaml.cs

```diff
@@ -210,8 +215,9 @@ private void PopulateDrives()
         /// </summary>
         private void PopulateDriveSpeeds()
         {
-            cmb_DriveSpeed.ItemsSource = _driveSpeeds;
-            cmb_DriveSpeed.SelectedItem = 8;
+            var values = UIElements.GetAllowedDriveSpeedsForMediaType(_currentMediaType);
+            cmb_DriveSpeed.ItemsSource = values;
+            cmb_DriveSpeed.SelectedIndex = values.Count / 2;
```

Comment:
Clarification question: Why use the middle value instead of something like the set value of 8? Is the middle value more preferable?

Reply:
Because I wasn't sure we can assume that 8 is always present, we should decide the best fit here.